sentiment_analysis.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **IMPORTANT**\n",
+    "For anything in this notebook to work, you need to install the ThirdAI package by signing up [here](https://www.thirdai.com/try-bolt/)."
```

Comment:
nit: To run this notebook, you need to download the ThirdAI docker container by...